C++/1 Learn the basics/1.5 Learn Basic Recursion/6 Factorial.cpp

```diff
@@ -0,0 +1,31 @@
+// https://bit.ly/3QugUVo
+
+#include<bits/stdc++.h>
+using namespace std;
+
+vector<long long> factorialNumbers(long long N)
+    {
+        // Write Your Code here
+        vector<long long> f;
+        f.push_back(1);
+        if(N == 1) return f;
+        f.push_back(2);
+        for(int i = 3; i < N; i++){
+            long long ans = i* f[i-2];
+            if(ans > N) break;
+            f.push_back(ans);
+        }
+        return f;
+    }
```

Comment:
Here Recursion isn't used!!
This is a iterative version of the question. As per the sheet you should have a recursive implementation.